sdk/python/feast/infra/offline_stores/file.py

```diff
@@ -1,6 +1,7 @@
 from datetime import datetime
 from typing import Callable, List, Optional, Union
 
+import dask.dataframe as dd
```

Comment:
If Dask is now always a requirement, why is the dependency added into extras?

Reply:
Fixed - added to the main requirements.